hack/make-rules/test-cmd-util.sh

```diff
@@ -5219,6 +5219,12 @@ runTests() {
 
     output_message=$(kubectl auth can-i list jobs.batch/bar -n foo --quiet 2>&1 "${kube_flags[@]}")
     kube::test::if_empty_string "${output_message}"
+    
+    output_message=$(kubectl auth can-i get pods --subresource=log 2>&1 "${kube_flags[@]}"; echo $?)
```

Comment:
This is still not testing your particular change. Can you add a test for your case, iow. one that will return non-zero when not allowed?

Reply:
Actually there is no way to test kubectl returning exit code 1, since the apiserver is in AlwaysAllow, how can we test this?

Reply:
Yeah, we'll have to live with a manual test. See https://github.com/kubernetes/kubernetes/pull/59579#discussion_r187638396

Reply:
@soltysh so what changes should I make? Since there is no way to test error exit 1 I think the commit is good as it is now.